semgrep/tests/e2e/targets/cli_test/multiple_annotations/multiple-annotations-bad.py

```diff
@@ -0,0 +1,8 @@
+# ruleid: basic-test
```

Comment:
why this file was called multiple-annotations-bad.py? Looks like it's actually the correct file, and the
file below (multiple-annotations.py) contains bad syntax

Reply:
it's also not even used in the test code so I wonder why it was added in the PR.

Reply:
fixed here I think: https://github.com/semgrep/semgrep/pull/10189